core/src/test/scala/org/apache/spark/util/FileAppenderSuite.scala

```diff
@@ -201,24 +201,29 @@ class FileAppenderSuite extends SparkFunSuite with BeforeAndAfter with Logging {
 
     // Make sure only logging errors
```

Comment:
For some reason these tests failed in one of my runs. And when they failed, they used leave the error level at "ERROR", which basically means all tests that ran after these didn't write any logs.